pennylane/hf/integrals.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+TThis module contains functions needed for computing integrals over basis functions.
+"""
+import numpy as np
+import autograd.numpy as anp
+from scipy.special import factorial2 as fac2
+
+
+def primitive_norm(l, alpha):
+    r"""Compute the normalization constant for a primitive Gaussian function.
+
+    A Gaussian function is defined as
+
+    .. math::
+
+        G = x^l y^m z^n e^{-\alpha r^2},
+
+    where :math:`l = (l, m, n)` defines the angular momentum quantum numbers, :math:`\alpha`
+    is the exponent and :math:`r = (x, y, z)` determines the center of the function. The
+    normalization constant for this function is computed as
+
+    .. math::
+
+        N(l, \alpha) = (\frac{2\alpha}{\pi})^{3/4} \frac{(4 \alpha)^{(l_x + l_y + l_z)/2}}
+        {(2\l_x-1)!! (2\l_y-1)!! (2\l_z-1)!!)^{1/2}}.
+
+    Args:
+        l (tuple[int]): angular momentum quantum numbers of the basis function
+        alpha (array[float]): exponent of the primitive Gaussian function
+
+    Returns:
+        n (array[float]): normalization coefficient
+
+    **Example**
+    >>> l = (0, 0, 0)
+    >>> alpha = np.array([3.425250914])
+    >>> n = gaussian_norm(l, alpha)
+    >>> print(n)
+    array([1.79444183])
+    """
+    lx, ly, lz = l
```

Comment:
Probably it is easier to change the notation in the docstring for consistency. For example:

> G = x^l y^m z^n e^{-\alpha r^2}  ==>> G = x^l_x y^l_y z^l_z e^{-\alpha r^2}